.github/workflows/spherical_geometry.yml

```diff
@@ -0,0 +1,48 @@
+name: Spherical Geometry CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Python Testing ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -e .[test]
+    - name: Test with pytest
+      run: |
+        pytest
+
+  doc_build:
```

Comment:
Why not use RTD PR builder?

Reply:
I removed `.travis.yml`.  I renamed the github action `blank.yml` to `spherical_geometry.yml`.